app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,11 @@
+class MessagesController < ApplicationController
+  def random_greeting
+    message = Message.order('RANDOM()').first
```

Comment:
You have done great so far, but there are some requirements you need to meet before moving to the next milestone.

-[ ] Your node module is missing two important dependencies `react-dom, babel-loader`, I'll suggest you go through the project lesson one more time and follow the guideline.